client/components/date-range/footer.tsx

```diff
@@ -26,7 +26,7 @@ const DateRangeFooter: FunctionComponent< Props > = ( {
 	return (
 		<div className="date-range__popover-footer">
 			<Button
-				className="date-range__cancel-btn"
+				className="date-range__cancel-btn is-secondary"
```

Comment:
This should be `variant="secondary"`, the class is a deprecated use

Reply:
Thanks for the feedback @mtias. Will address in a follow up PR.